packages/talos_certifier/src/services/decision_outbox_service.rs

```diff
@@ -123,21 +112,35 @@ impl SystemService for DecisionOutboxService {
                 headers,
                 message: decision_message,
             } = decision_channel_message;
-            tokio::spawn(async move {
-                match DecisionOutboxService::save_decision_to_xdb(&datastore, &decision_message).await {
-                    Ok(decision) => {
-                        if let Err(publish_error) = DecisionOutboxService::publish_decision(&publisher, &decision, headers).await {
-                            error!(
-                                "Error publishing message for version={} with reason={:?}",
-                                decision.version,
-                                publish_error.to_string()
-                            );
+            tokio::spawn({
+                let decision_headers = DecisionHeaderBuilder::with_additional_headers(headers.into()).add_meta_headers(DecisionMetaHeaders::new(
+                    1_u64, // major version of decision message
```

Comment:
You might want to keep it somewhere like a constant or "static field", visible. Here it is a bit buried in the code logic.

Reply:
Thanks for spotting that. Will update it today